coco/master.py

```diff
@@ -37,6 +46,13 @@
     # create a unique name for this task: <process ID>-<POSIX timestamp>
     name = f"{os.getpid()}-{time.time()}"
 
+    # increment prometheus counter
+    cnt = COUNTERS.get(endpoint, None)
+    if cnt is None:
+        logger.error(f"No prometheus metric for endpoint {endpoint}.")
```

Comment:
Should this result in `exit(1)`?

Reply:
I wasn't sure what the appropriate thing to do would be. Seems like this should never happen unless something weird is going on, so maybe `exit(1)` would be the right thing to do.

Reply:
Yup. If we end up here, we want to know for sure

Reply:
Sorry, I missed that the first time: try/except will be faster here than `if`, since we don't expect this to go wrong.

Reply:
oh yeah, that's a good point